.travis.yml

```diff
@@ -1,7 +1,6 @@
 sudo: false
 language: perl
 perl:
-  - "5.22"
```

Comment:
Travis doesn't have 5.22, so it just runs 5.14 iirc, which we already test
On 31 Oct 2015 19:18, "Michael D. Stemle, Jr." <notifications@github.com>
wrote:

> In .travis.yml
> <https://github.com/net-amqp-rabbitmq/net-amqp-rabbitmq/pull/90#discussion_r43575205>
> :
>
> > @@ -1,7 +1,6 @@
> >  sudo: false
> >  language: perl
> >  perl:
> > -  - "5.22"
>
> We need to add 5.22 back in, no?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/net-amqp-rabbitmq/net-amqp-rabbitmq/pull/90/files#r43575205>
> .
>